.pre-commit-config.yaml

```
@@ -56,10 +56,10 @@ repos:
     rev: v0.9.0
     hooks:
       - id: unasyncd
-        additional_dependencies: [ "ruff" ]
+        additional_dependencies: [ "ruff>=0.14.10,<0.15" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)